### ISSUE TITLE
Russian: translation

### ISSUE THREAD

**Issue by marc-medley** [nutritionfactsorg/daily-dozen-localization#9]
This thread has been setup for `.tsv` Russian translation uploads to be processed for the Android and iOS Daily Dozen applications.

**Comment by salitrov** [comment]
Attaching the new Russian translation. Feel welcome to share the Additions file and we'll take care of that. Thank you.
[Russian Daily Dozen App Translation - Russian_ru.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/7375114/Russian.Daily.Dozen.App.Translation.-.Russian_ru.txt)

**Comment by marc-medley** [comment]
@salitrov Thanks. The Russian_ru.additions.tsv file is located in the `daily-dozen-localization` repository [Languages/Russian/tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/Russian_ru.additions.tsv) directory.

For convenience, each language directory is linked to the [main dashboard table](https://github.com/nutritionfactsorg/daily-dozen-localization#dashboard-) in the `Languages` column.  Where applicate, a `*.additions.tsv` have been placed for each respective language in the `tsv` subfolder.

**Comment by salitrov** [comment]
Uploading the new additions and appstore sheets. Thank you for your support.

[Russian Additions Sheet - Sheet1.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/7435291/Russian.Additions.Sheet.-.Sheet1.txt)

[Russian Appstore Sheet - Sheet1.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/7435292/Russian.Appstore.Sheet.-.Sheet1.txt)

**Comment by marc-medley** [comment]
@salitrov Processing and source level checks have been completed for [Russian](https://github.com/nutritionfactsorg/daily-dozen-localization/tree/master/Languages/Russian).

The current, updated TSV files to use for any Russian translation updates are:

* [Russian_ru.app.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/Russian_ru.app.tsv)
* [Russian_ru.store.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/Russian_ru.store.tsv)

Separately, I've sent an email with some information regarding pre-release software distribution for  translation review & test.

**Comment by marc-medley** [comment]
I removed the square brackets `[]` since the brackets since that brackets were not intended to be part of the translation. Please advise if any other change would be needed with out the brackets. 

Before: `Добавлена языковая поддержка [Русский]`
After: `Добавлена языковая поддержка Русский`

**Comment by salitrov** [comment]
Thank you, Marc. They have been removed. I was told her review is finished and I am attaching the latest versions of the sheets.

Also, it looks like more changes are incoming as we are changing the Brussels sprout recommendation from 1/4 cup to 1/2 cup. More on that should be coming soon.

[Russian Daily Dozen App Translation - Russian_ru.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/7989630/Russian.Daily.Dozen.App.Translation.-.Russian_ru.txt)
[Russian Additions Sheet - Sheet1.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/7989631/Russian.Additions.Sheet.-.Sheet1.txt)
[Russian Appstore Sheet - Sheet1.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/7989632/Russian.Appstore.Sheet.-.Sheet1.txt)

**Comment by salitrov** [comment]
Actually, she just told me she wasn't able to review the App Store part within the app because she's not sure how. How is she able to do that? Thank you.

**Comment by marc-medley** [comment]
The App Store text will appear on the App Store web page. It's not part of the application itself.

The `daily-dozen-localization` dashboard columns `Google Play` and `App Store Region`. Provides links to the pages where the translatation will be visible when published.

--> [`daily-dozen-localization` dashboard](https://github.com/nutritionfactsorg/daily-dozen-localization#dashboard-)

The Russian pages are here:

* [Google Play](https://play.google.com/store/apps/details?id=org.nutritionfacts.dailydozen&hl=ru)
* [App Store](https://apps.apple.com/ru/app/dr-gregers-daily-dozen/id1060700802)

I'll check if a pre-publication viewing of the store page is possible.

**Comment by salitrov** [comment]
Thank you. I think we were mostly just curious about being able to review it before publishing, but no worry if we can only after it's published. We'll just make sure the sheet is all good and check it again once it's published and see if further changes are needed.

**Comment by salitrov** [comment]
She has reviewed the App Store sheet and some changes were made. Otherwise, it looks like it should be good to go for publishing for now! Let me know if there are any other updates or needs. Thank you.

[Russian Appstore Sheet - Sheet1.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/8036110/Russian.Appstore.Sheet.-.Sheet1.txt)

**Comment by salitrov** [comment]
Sorry about this. They just sent me more updates on this doc too.

[Russian Daily Dozen App Translation - Russian_ru.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/8043198/Russian.Daily.Dozen.App.Translation.-.Russian_ru.txt)

**Comment by marc-medley** [comment]
@salitrov  This following spreadsheets are up-to-date relative to the application code base. If possible, please have the translators do any updates using these current repository TSV spreadsheet versions. Thanks.

**Russian**

* [Russian/tsv/Russian_ru.app.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/Russian_ru.app.tsv)
* [Russian/tsv/Russian_ru.store.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/Russian_ru.store.tsv)

**Comment by marc-medley** [comment]
There is clipping on the small iPhone screen on the Preferences screen. 

I'm looking at adjusting the layout on the software side since this is an issue with several of the non-English languages.

_Russian_
![Preferences ru](https://user-images.githubusercontent.com/15098151/157847760-7d2c004c-8128-4197-af15-7db641ca6c0a.jpg)

_English_
![Preferences en](https://user-images.githubusercontent.com/15098151/157847295-d3954e3a-7bed-44da-bf67-05e599404970.png)

**Comment by salitrov** [comment]
Hi Marc,

It (row 369) has been shortened. New file attached here. Let me know if there's anything else we can do.
[Russian Daily Dozen App Translation - Russian_ru.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/8306941/Russian.Daily.Dozen.App.Translation.-.Russian_ru.txt)

**Comment by marc-medley** [comment]
Fixed in pre-release build v3.2.11(29).

![ru_info_v3 2_snip](https://user-images.githubusercontent.com/15098151/162289882-0314ea29-c44a-47e6-83b7-2bb7715e7546.png)

---

![ru_settings_v3 2 11(29)_snip](https://user-images.githubusercontent.com/15098151/162290359-774a3a77-63a6-46d6-8072-eab9769b7359.png)

**Comment by marc-medley** [comment]
@salitrov The translators can now review the live App Store page for iOS Daily Dozen with Russian:

* https://apps.apple.com/ru/app/dr-gregers-daily-dozen/id1060700802

iPhone screenshots are attached below in a zip file. The iPhone screenshots can be replaced with marketing images when available. Image specification for the two sets: 1242 x 2688 pixels (portrait) and 1242 x 2208 pixels (portrait).

[Russian_ru_Screenshots_ios.zip](https://github.com/nutritionfactsorg/daily-dozen-localization/files/8548113/Russian_ru_Screenshots_ios.zip)

**Comment by salitrov** [comment]
Thank you, Marc. The translator/reviewer had a question about the book titles in the App Store description:
Can I use italic? If not, I will use double quotes. I thought Italic would be applied to the final formatting of book names for all languages, but it does not seem to be the case. Right now book names do not look good w/o any formatting, especially How not to Die ends with an exclamation mark in Russian.

What would you recommend?

**Comment by marc-medley** [comment]
@salitrov Perhaps the punctuation marks of the English reference and other product page translations could be improved as well?  Some comments below...

> Can I use italic? 

All product page text components are plain text \*without\* any style formatting such as _italics_ or **bold**. Plain text is used for both Android and Apple product page text content. Translations will need to use plain text characters, such as quotes, for elements like book titles as appropriate for that language.

A note about plain text has now been added to the [Technical Note: Application Product Page](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Documentation/TechNote_AppProductPage.md#text-components-) document.

> ...will use double quotes.

All [international (UTF-8) quotation marks](https://en.wikipedia.org/wiki/Quotation_mark) and other punctuation marks are supported in the product page plain text.

> Right now book names do not look good w/o any formatting, especially … in Russian.

Also, the baseline `English_US_en.store.tsv` reference currently does not have any quotes around the book titles.

* [English_US_en.store.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/English_US/tsv/English_US_en.store.tsv)
* [App Store: Dr. Greger’s Daily Dozen](https://apps.apple.com/us/app/dr-gregers-daily-dozen/id1060700802)

> What would you recommend?

Review and, if applicable, upload an updated [English_US_en.store.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/English_US/tsv/English_US_en.store.tsv). If English_US remains without quotes around the book titles, then add some translation guidance to use quotes around book titles as applicable to the `base_note` column.

See: [English_US: baseline reference #19](https://github.com/nutritionfactsorg/daily-dozen-localization/issues/19)

Upload an updated Russian version of [Russian_ru.store.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/Russian_ru.store.tsv)  as a `.txt` or `.zip` file to this thread.

Upload other updated `*.store.tsv` as a `.txt` or `.zip` file to their corresponding "[Language]: translation" Issues thread. 

Let me know when the updates are uploaded. When notified, I'll update the `daily-dozen-localization` repository which @slavick and I will then use to update the product pages.

As an aside, the Russian feedback has so far been good:

> Спасибо разработчикам за добавление русского языка 🤝 (5-stars)

**Comment by salitrov** [comment]
Thank you, Marc. That sounds good to add quotations. Both of these Russian sheets have been updated (including quotations for book titles) and they also suggested swapping out the screenshot in the app store with the chickpeas on it since the translation was updated.

[Russian Daily Dozen App Translation - Russian_ru.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/8688777/Russian.Daily.Dozen.App.Translation.-.Russian_ru.txt)

[Russian Appstore Sheet - Sheet1.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/8688778/Russian.Appstore.Sheet.-.Sheet1.txt)

**Comment by salitrov** [comment]
Hey Marc, just checking in to see if this was updated accordingly. The chickpeas screenshot looks the same as before. I was asked by the translator to let her know once it was updated. Thanks for considering.

**Comment by marc-medley** [comment]
@salitrov A TestFlight  beta version, with Russian updates, is now being review by the Apple App Store. The App Store TestFlight process will automatically notify registered testers via email. I will also update you on when a test version is approved.

**Comment by marc-medley** [comment]
@salitrov The `changeset01` tsv files contain some addition lines which need translation to support the addition of Google Firebase Analytics to the application, translator recognition, and the cruciferous servings update. 

* [Russian_ru.app.changeset01.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/v01/Russian_ru.app.changeset01.tsv)
* [Russian_ru.store.changeset01.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/v01/Russian_ru.store.changeset01.tsv)

**Comment by salitrov** [comment]
Here is the Russian update. The translator requested if they can review them within the app/page before they go live. Thank you.

[Russian_ru.changeset01 - Russian_ru.changeset01.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/9269460/Russian_ru.changeset01.-.Russian_ru.changeset01.txt)
[Russian_ru.appstore.changeset01 - Russian_ru.appstore.changeset01.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/9269461/Russian_ru.appstore.changeset01.-.Russian_ru.appstore.changeset01.txt)

**Comment by salitrov** [comment]
Here are the Russian translation updates. Let me know if anything else is needed. Thanks!

[Russian_ru.changeset01 - Russian_ru.changeset01.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/9499741/Russian_ru.changeset01.-.Russian_ru.changeset01.txt)
[Russian_ru.appstore.changeset01 - Russian_ru.appstore.changeset01.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/9499742/Russian_ru.appstore.changeset01.-.Russian_ru.appstore.changeset01.txt)

**Comment by marc-medley** [comment]
@salitrov 

Please note that the Apple App Store does not provide for a "preview" like the Apple TestFlight does provide for the app. Please have the translator review the current app place. This page should be up-to-date with all the received changes.

* [Russian Apple App Store page](https://apps.apple.com/ru/app/dr-gregers-daily-dozen/id1060700802)

Also, build Build 3.3.1 (1) was approved by Apple for beta testing on Oct 7. All translators on the Apple testing list should have recieved an automatic TestFlight notification from Apple at that time.

**Comment by marc-medley** [comment]
@salitrov Russian translation Change Set v03 baseline is available here:

* [Russian/tsv/v03](https://github.com/nutritionfactsorg/daily-dozen-localization/tree/master/Languages/Russian/tsv/v03)

**Comment by marc-medley** [comment]
@salitrov The text needed to support the data export and import on the iPhone Daily Dozen app is ready for translation. Thanks.

* Translation Spreadsheet location: [Russian/tsv/v02](https://github.com/nutritionfactsorg/daily-dozen-localization/tree/master/Languages/Russian/tsv/v02)

**Comment by marc-medley** [comment]
@salitrov Links for files currently needing/awaiting Russian translation:

* [Russian_ru.app.changeset02.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/v02/Russian_ru.app.changeset02.tsv)
* [Russian_ru.app.changeset03.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/v03/Russian_ru.app.changeset03.tsv)
* [Russian_ru.store.changeset03.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/v03/Russian_ru.store.changeset03.tsv)

**Comment by salitrov** [comment]
Latest Russian updates translated. Feel welcome to let me know if anything else is needed.

[Russian_ru.app.changeset02 - Russian_ru.app.changeset02.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/11960418/Russian_ru.app.changeset02.-.Russian_ru.app.changeset02.txt)
[Russian_ru.app.changeset03 - Russian_ru.app.changeset03.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/11960419/Russian_ru.app.changeset03.-.Russian_ru.app.changeset03.txt)
[Russian_ru.store.changeset03 - Russian_ru.store.changeset03.txt](https://github.com/nutritionfactsorg/daily-dozen-localization/files/11960420/Russian_ru.store.changeset03.-.Russian_ru.store.changeset03.txt)

**Comment by marc-medley** [comment]
The Russian changeset02/changeset03 updates have completed intake processing and have been merged upto the latest v05. The current files for any future updates are now:

- [Russian/tsv/v05_Peanuts/Russian_ru.app.v05.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/v05_Peanuts/Russian_ru.app.v05.tsv)
- [Russian/tsv/v05_Peanuts/Russian_ru.store.v05.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/v05_Peanuts/Russian_ru.store.v05.tsv)

There are two items which were not covered in the last submission, which can be seen in the [Russian_ru.app.v05.tsv](https://github.com/nutritionfactsorg/daily-dozen-localization/blob/master/Languages/Russian/tsv/v05_Peanuts/Russian_ru.app.v05.tsv) file.

- line 331 `faq_title`, `Frequently Asked Questions`, `Часто задаваемые вопросы`
- line 198 `dozeNuts.Variety.Text.7`, `Peanuts`, `Арахис`
